src/orders/components/OrderProductAddDialog/OrderProductAddDialog.tsx

```diff
@@ -363,9 +363,7 @@ const OrderProductAddDialog: React.FC<OrderProductAddDialogProps> = props => {
                             </div>
                           </TableCell>
                           <TableCell className={classes.textRight}>
-                            {variant?.channelListings[0]?.price && (
-                              <Money money={variant.channelListings[0].price} />
-                            )}
+                            <OrderPriceLabel pricing={variant.pricing} />
```

Comment:
```suggestion
                            <OrderPriceLabel pricing={variant?.pricing} />
```
I'm pretty sure optional chaining should stay here, because variant field tend to be undefined in many cases

Reply:
If `product.variants` is empty then the whole table row is not rendered